docs/data/debugger.md

```diff
@@ -3,20 +3,36 @@ title: Debugging Analytics
 description: Troubleshoot and debug any data errors that may occur in the instrumentation process.
 ---
 
-Data validation is a critical step in the instrumentation process. Amplitude lets validate your event data via Amplitude's [User Lookup](#user-lookup) or using the [Instrumentation Explorer](#instrumentation-explorer) Chrome extension.
+Data validation is a critical step in the instrumentation process. Amplitude lets validate your event data by:
+
+- [Ingestion Debugger](#ingestion-debugger)
+- [User Lookup](#user-lookup)
+- [Instrumentation Explorer](#instrumentation-explorer) Chrome extension
 
 ### Before you begin: Instrument your events
 
-If you haven't instrumented any events, Amplitude's servers don't receive data, and no data is available in Amplitude itself. Amplitude strongly recommends that you create a test app to test and validate your first event data. See the [Quick Start Guide](https://help.amplitude.com/hc/sections/201146908) for more instructions and advice on getting started with Amplitude.
+**If you haven't instrumented any events, Amplitude's servers don't receive data, and no data is available in Amplitude itself**. Amplitude strongly recommends that you create a test app to test and validate your first event data. See [Plan your set up](../../analytics/plan-your-set-up/) for more instructions and advice on getting started with Amplitude.
+
+## Ingestion debugger
```

Comment:
How about putting some screenshot here?

Reply:
It is also not recommended to add screenshots when plain text can explain everything. Here are some reasons I don't want to add screenshots here:
- The navigation to ingestion debugger is not complex
- each chart of ingestion debugger has a title and is self-explanatory
- Another problem to add screenshots here is that the length of ingestion debugger is very long, if I add a screenshot of it as a whole, it might case this page less visually attractive
- If add four charts separately, it still takes up lot of content space in the page